alts/worker/runners/base.py

```diff
@@ -102,11 +102,13 @@ def __init__(self, task_id: str, dist_name: str,
         self._logger = self.init_test_task_logger(task_id, dist_arch)
         self._task_log_file = None
         self._task_log_handler = None
-        self._work_dir = None
+        self._work_dir = None  # type: typing.Optional[Path]
         self._artifacts_dir = None
-        self._inventory_file_path = None
-        self._integrity_tests_dir = None
-        self._class_resources_dir = os.path.join(RESOURCES_DIR, self.TYPE)
+        self._inventory_file_path = None  # type: typing.Optional[Path]
```

Comment:
Why do you use type annotations in comments?

Reply:
for IDE